patroni/config_generator.py

```diff
@@ -287,7 +287,7 @@ def _set_pg_params(self, cur: Union['cursor', 'Cursor[Any]']) -> None:
 
         :param cur: connection cursor to use.
         """
-        cur.execute("SELECT name, current_setting(name) FROM pg_settings "
+        cur.execute("SELECT name, pg_catalog.current_setting(name) FROM pg_catalog.pg_settings "
```

Comment:
can we avoid this unrelated change?

Reply:
No, I want to explicitly call `schema.func_name()` and want to reuse mocks :)